gadgetchains/PopPHP/FW/1/gadgets.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Pop\Mail\Transport\Smtp
+{
+    class EsmtpTransport
```

Comment:
Does not exist.

Reply:
maybe because the below reason or `composer install`?